web/profiles/custom/yalesites_profile/modules/custom/ys_themes/config/install/ys_themes.component_overrides.yml

```diff
@@ -0,0 +1,77 @@
+callout:
+  field_style_color:
+    values:
+      blue-yale: 'Blue'
+      gray-700: 'Gray'
+      beige: 'Beige'
```

Comment:
Semantics question. What do you think about making in an array instead of a set of properties? Doesn't really matter as long as we are consistant. Just popped in mind.
```yaml
    values:
      - blue-yale: 'Blue'
      - gray-700: 'Gray'
      - beige: 'Beige'
```

Reply:
Good point, but, yes, if we're consistent I think it's fine to leave as is for now unless we need it to